### PR TITLE
Update README: Replace API Key with Auth Token and Reflect Changes in…

### PR DESCRIPTION
**Pull request by sbrij001**
… YAML File

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by sbrij001** [comment]
@bnb Will you review this PR?

File: README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ jobs:
       - name: 'Sending SMS Notification'
         uses: twilio-labs/actions-sms@v1
         with:
-          fromPhoneNumber: '+12345678900' # alternatively, use a Repository Secret
-          toPhoneNumber: '+12345678900' # alternatively, use a Repository Secret
+          fromPhoneNumber: ${{ secrets.FROM_PHONE_NUMBER }} # alternatively, use a Repository Secret
+          toPhoneNumber: ${{ secrets.TO_PHONE_NUMBER }} # alternatively, use a Repository Secret
```
Comment (bnb) [inline README.md:36]: the comments from the original line don't make sense anymore since that's what we're doing

File: README.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ Send an SMS from GitHub Actions.
 ## Prerequisites
 
 - A Twilio Account. [Sign up for free](https://www.twilio.com/try-twilio)
-- A [Twilio API Key and Secret](https://www.twilio.com/docs/iam/keys/api-key)
+- A [Twilio Auth Token](https://www.twilio.com/docs/iam/api/authtoken)
```
Comment (bnb) [inline README.md:8]: Might want to include a strong recommendation for a verified number

**Review by bnb — COMMENTED** [review]
some notes

File: README.md (hunk 2 above)
Comment (sbrij001) [reply README.md:8]: Linked to regulatory requirement FAQ

**Comment by bnb** [comment]
Should also be merged with #73

**Review by bnb — APPROVED** [review]

**Comment by ayyrickay** [comment]
@sbrij001 merging these in to close this out

**Pull request merged**